contracts/axelar-gateway/src/test_auth.rs

```diff
@@ -646,3 +646,88 @@ fn rotate_signers_fail_duplicated_signers() {
         );
     });
 }
+
+#[test]
+fn epoch_by_signers_hash() {
```

Comment:
the gateway tests are enough since they cover these methods. we don't need extra unit tests here

Reply:
I see. make sense. updated.